Worker is not found on docker start

```
[0] node:internal/modules/cjs/loader:998
[0]   throw err;
[0]   ^
[0]
[0] Error: Cannot find module '/app/background/worker/index.js'
[0]     at Module._resolveFilename (node:internal/modules/cjs/loader:995:15)
[0]     at Module._load (node:internal/modules/cjs/loader:841:27)
[0]     at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:81:12)
[0]     at node:internal/main/run_main_module:23:47 {
[0]   code: 'MODULE_NOT_FOUND',
[0]   requireStack: []
[0] }
```

--- a/Dockerfile
+++ b/Dockerfile
@@ -22,4 +22,4 @@ RUN npx prisma generate
 
 EXPOSE 3000
 
-CMD ["node_modules/.bin/concurrently", "node background/worker/index.js", "node server.js"]
+CMD ["node_modules/.bin/concurrently", "node background/utils/worker/index.js", "node server.js"]

